docs: refactor the docs associated with Logger (#6415)
apache/apisix: docs/en/latest/batch-processor.md

The only mandatory parameter to create a batch processor is a function. The function will be executed when the batch reaches the max size
or when the buffer duration exceeds.

| Name             | Type    | Requirement | Default | Valid   | Description                                                  |
| ---------------- | ------- | ----------- | ------- | ------- | ------------------------------------------------------------ |

> | batch_max_size   | integer | optional    | 1000    | [1,...] | Sets the maximum number of logs sent in each batch. When the number of logs reaches the set maximum, all logs will be automatically pushed to the HTTP/HTTPS service. |

Reviewer: Why we remove `name` attr desc here ?

> | batch_max_size   | integer | optional    | 1000    | [1,...] | Set the maximum number of logs sent in each  batch. When the number of logs reaches the set maximum, all logs will be  automatically pushed to the HTTP/HTTPS service. |

Reviewer 2: Remove extra spaces. Please double check all texts.
```suggestion
| batch_max_size   | integer | optional    | 1000    | [1,...] | Set the maximum number of logs sent in each batch. When the number of logs reaches the set maximum, all logs will be automatically pushed to the HTTP/HTTPS service. |
```

> | inactive_timeout | integer | optional    | 5       | [1,...] | The maximum time to refresh  the buffer (in seconds). When the maximum refresh time is reached, all logs  will be automatically pushed to the HTTP/HTTPS service regardless of whether the number of logs in the buffer  reaches the maximum number set. |

Reviewer 2: Remove extra spaces. 
```suggestion
| inactive_timeout | integer | optional    | 5       | [1,...] | The maximum time to refresh the buffer (in seconds). When the maximum refresh time is reached, all logs will be automatically pushed to the HTTP/HTTPS service regardless of whether the number of logs in the buffer reaches the maximum number set. |
```

> | Name             | Type    | Requirement | Default | Valid   | Description                                                  |
> | ---------------- | ------- | ----------- | ------- | ------- | ------------------------------------------------------------ |
> | batch_max_size   | integer | optional    | 1000    | [1,...] | Set the maximum number of logs sent in each  batch. When the number of logs reaches the set maximum, all logs will be  automatically pushed to the HTTP/HTTPS service. |
> | inactive_timeout | integer | optional    | 5       | [1,...] | The maximum time to refresh  the buffer (in seconds). When the maximum refresh time is reached, all logs  will be automatically pushed to the HTTP/HTTPS service regardless of whether the number of logs in the buffer  reaches the maximum number set. |
> | buffer_duration  | integer | optional    | 60      | [1,...] | Maximum age in seconds of the  oldest entry in a batch before the batch must be processed. |
> | max_retry_count  | integer | optional    | 0       | [0,...] | Maximum number of retries  before removing from the processing pipe line. |
> | retry_delay      | integer | optional    | 1       | [0,...] | Number of seconds the process  execution should be delayed if the execution fails. |

Reviewer 3: ```suggestion
| Name             | Type    | Requirement | Default | Valid   | Description                                                  |
| ---------------- | ------- | ----------- | ------- | ------- | ------------------------------------------------------------ |
| batch_max_size   | integer | optional    | 1000    | [1,...] | Sets the maximum number of logs sent in each  batch. When the number of logs reaches the set maximum, all logs will be automatically pushed to the HTTP/HTTPS service. |
| inactive_timeout | integer | optional    | 5       | [1,...] | The maximum time to refresh  the buffer (in seconds). When the maximum refresh time is reached, all logs will be automatically pushed to the HTTP/HTTPS service regardless of whether the number of logs in the buffer reaches the maximum number set. |
| buffer_duration  | integer | optional    | 60      | [1,...] | Maximum age in seconds of the oldest entry in a batch before the batch must be processed. |
| max_retry_count  | integer | optional    | 0       | [0,...] | Maximum number of retries before removing the entry from the processing pipeline when an error occurs. |
| retry_delay      | integer | optional    | 1       | [0,...] | Number of seconds the process execution should be delayed if the execution fails. |
```

> | buffer_duration  | integer | optional    | 60      | [1,...] | Maximum age in seconds of the  oldest entry in a batch before the batch must be processed. |
> | max_retry_count  | integer | optional    | 0       | [0,...] | Maximum number of retries  before removing from the processing pipe line. |
> | retry_delay      | integer | optional    | 1       | [0,...] | Number of seconds the process  execution should be delayed if the execution fails. |

Reviewer 2: Remove extra spaces. 
```suggestion
| buffer_duration  | integer | optional    | 60      | [1,...] | Maximum age in seconds of the oldest entry in a batch before the batch must be processed. |
| max_retry_count  | integer | optional    | 0       | [0,...] | Maximum number of retries before removing from the processing pipe line. |
| retry_delay      | integer | optional    | 1       | [0,...] | Number of seconds the process execution should be delayed if the execution fails. |
```
